README.md

```diff
@@ -177,7 +177,7 @@ Before running other scripts, stop this one by CTRL+C (or wait, it will shutdown
 For turning the motor in different modes, you will need 5 terminals and in all of them execute:
 ```bash
 sudo -i
-source ~/.bashrc
+source /home/YOUR_USER/.bashrc
```

Comment:
This didn't work previously because when you run `sudo -i`, it changes the user of the terminal to `sudo`.

Reply:
I see.

BTW: why do we have to use sudo at all? Can't we use the users non-sudo permissions?

Reply:
I'm not an expert on ethercat but when I googled it, almost all suggestions involved `sudo`. I don't like it either. I guess it's a security risk to allow anybody to access the ethernet port.

At least there is another option in the README, it's just a bit of extra work. https://github.com/synapticon/synapticon_ros2_control/blob/main/README.md#running-without-sudo-optional